_posts/2017-04-30-php.md

```diff
@@ -4,8 +4,10 @@ title: "PHP"
 code: |-
     echo .1 + .2; 
     var_dump(.1 + .2);
+    var_dump(bc_add(.1, .2, 1));
```

Comment:
It looks like the method is `bcadd` (no underscore). https://www.php.net/manual/en/function.bcadd.php